dbms/src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -2763,8 +2768,27 @@ void MergeTreeData::movePartitionToVolume(const ASTPtr & partition, const String
             if (part->disk->getName() == disk->getName())
                 throw Exception("Part " + part->name + " already on volume '" + name + "'", ErrorCodes::UNKNOWN_DISK);
 
-    if (!movePartsToSpace(parts, std::static_pointer_cast<const DiskSpace::Space>(volume)))
-        throw Exception("Cannot move parts because moves are manually disabled.", ErrorCodes::ABORTED);
+    parts.erase(std::remove_if(parts.begin(), parts.end(), [&](auto part_ptr) {
```

Comment:
Need to fix the style of lambda and maybe it would be more obvious to write the for loop with named flags like `found_part_on_another_disk = false`?

Reply:
@alesapin 
> It's very easy to test with the integration test. Just move one part with `ALTER TABLE MOVE PART` and then try to move partition.

3debdc211904eb0b5fe6cca1a4af99d8db635892

Reply:
I don't get the idea.
First, the code would be pretty complicated, like this:
```
    all_parts_already_on_volume = true;
    for (const auto & part : parts)
    {
        bool part_already_on_volume = false;
        for (const auto & disk : volume->disks)
        {   
            if (part_ptr->disk->getName() == disk->getName())
            {   
                part_already_on_volume = true;
                break;
            }
        }
        if (!part_already_on_volume)
        {   
            all_parts_already_on_volume = false;
            break;
        }
    }
```
And I will need vector of parts anyway, because I pass them to `movePartsToSpace`.